app/uk/gov/hmrc/exports/routines/RoutineRunnerModule.scala

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 
-package uk.gov.hmrc.exports.migrations
+package uk.gov.hmrc.exports.routines
 
 import play.api.inject._
 
-class ExportsMigrationModule extends SimpleModule(bind[MigrationRunner].toSelf.eagerly())
+class RoutineRunnerModule extends SimpleModule(bind[RoutineRunner].toSelf.eagerly())
```

Comment:
I can't find the new Module being registered in `application.conf`

Reply:
Line 55